guides/common/modules/proc_synchronizing-a-custom-repository.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Synchronizing_a_Custom_Repository_{context}"]
+= Synchronizing a Custom Repository
+
+Use this procedure to synchronize content from a custom repository on a connected {ProjectServer} and a disconnected {ProjectServer} via {ISS} (ISS) Network Sync.
+
+Follow the procedure for the connected {ProjectServer} before completing the procedure for the disconnected {ProjectServer}.
+
+.Connected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click on the custom product.
+. Click on the custom repository.
+. Copy the *Published At:* URL.
+. Continue with the procedure on disconnected {ProjectServer}.
+
+.Disconnected {ProjectServer}
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Create your custom product with the following:
+* Upstream URL: Paste the link that you copied earlier.
+* On the connected {ProjectServer}, download the file `https:///pub/katello-server-ca.crt` using `curl` or `wget`.
```

Comment:
I'd move this step to connected section. Also, we could demonstrate a concrete command, like 
```
# curl [opts] https://{foreman-example-com}/pub/katello-server-ca.crt
```
I imagine that the curl needs verification with server certificates? @jeremylenz